### ISSUE TITLE
Crash in [@ libc.so@0x4e44c | libc.so@0x4e41c | libart.so@0x55d5f0 | libbase.so@0x13be4] while editing GitLab account settings

### ISSUE THREAD

**Issue by D4v1dH03** [mozilla-mobile/fenix#22689]
Maybe Fission related. (DOMFissionEnabled=1)

Crash report: https://crash-stats.mozilla.org/report/index/9c38e31e-9529-46e9-bce2-dd9f90211203

Reason: ```SIGABRT```

Top 10 frames of crashing thread:
```
0 libc.so libc.so@0x4e44c 
1 libc.so libc.so@0x4e41c 
2 libart.so libart.so@0x55d5f0 
3 libbase.so libbase.so@0x13be4 
4 libbase.so libbase.so@0x12fb4 
5 libart.so libart.so@0x2f9894 
6 libart.so libart.so@0x38c260 
7 libart.so libart.so@0x387784 
8 libart.so libart.so@0x5b2ab4 
9 libart.so libart.so@0x3bd95c 
```

Steps to reproduce:
Log into any Gitlab personal account and navigate to account preferences by clicking the top right corner pull down

And oops, it crashes at any time somtimes *right* after navigating to the preferences pages of GitLab

**Comment by D4v1dH03** [comment]
Same happens intermittently during webpage discord login too

**Comment by makotokato** [comment]
@D4v1dH03 could you reproduce this on 97 beta?  We have fixed related issue (https://bugzilla.mozilla.org/show_bug.cgi?id=1747116)

**Comment by D4v1dH03** [comment]
Confirmed fixed in latest Nightly (98.0a1), but since I don't have 97 beta installed on my phone so I am not sure
